frontend/src/components/Navbar.vue

```diff
@@ -58,7 +58,7 @@
                                     </a>
                                     <a @click="locale = 'zh_CN'; $i18n.locale = locale; dropdownVisible = false"
                                        class="cursor-pointer block px-4 py-2 hover:bg-base-300 dark:hover:bg-base-dark-400 transition-colors flex gap-2 items-center">
-                                        <p class="flex-1 pr-2">中文 (中国)</p>
+                                        <p class="flex-1 pr-2">中文（中国大陆）</p>
```

Comment:
Please do not add these unrelated changes, this is normal for Countries / Regions dropdowns

Reply:
With reference to the language list of Minecraft and Microsoft/Xbox.

Reply:
It really doesn't matter, this is an unrelated change. On the other note, pretty much most regions dropdowns use China (e.g. Discord), this isn't a nationality dropdown so there isn't an enticing reason to clarify the difference.